### ISSUE TITLE
Dragging refresh indicator far enough triggers scroll glow

### ISSUE THREAD

**Issue by Veeksi** [peng8350/flutter_pulltorefresh#547]
## Issue
So when the refresh indicator is dragged too far, a blueish scroll glow indicator shows up which is unwanted behaviour. To reproduce this issue you can use this simple code snippet below.

## GIF from this issue
https://i.imgur.com/lEV91vl.gif

```
class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: MyHomePage(),
    );
  }
}

class MyHomePage extends StatelessWidget {
  MyHomePage({Key? key}) : super(key: key);
  
  final RefreshController _refreshController = RefreshController();

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: const Text('Scroll glow issue'),
      ),
      body: SmartRefresher(
        controller: _refreshController,
        child: ListView.builder(
            shrinkWrap: true,
            itemCount: 20,
            itemBuilder: (context, i) {
              return const Card(
                child: Text('Test'),
              );
            }),
      ),
    );
  }
}
```

## Used device
Oneplus Nord 2
Android 11

## Flutter doctor -v

$ flutter doctor -v
[√] Flutter (Channel stable, 2.5.3, on Microsoft Windows [Version 10.0.19042.1288], locale en-FI)
    • Flutter version 2.5.3 at C:\src\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 18116933e7 (4 weeks ago), 2021-10-15 10:46:35 -0700
    • Engine revision d3ea636dc5
    • Dart version 2.14.4

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\XXXXX\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java      
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6842174)   
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.11.3)  
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.11.31702.278
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 4.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6842174)

[√] Android Studio (version 2020.3)
    • Android Studio at C:\Program Files\Android\Android Studio Preview
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[√] VS Code (version 1.62.1)
    • VS Code at C:\Users\XXXXX\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.28.0

[√] Connected device (4 available)
    • DN2103 (mobile)   • ZXMN7H7LZX7XNZH6 • android-arm64  • Android 11 (API 30)
    • Windows (desktop) • windows          • windows-x64    • Microsoft Windows [Version 10.0.19042.1288]
    • Chrome (web)      • chrome           • web-javascript • Google Chrome 95.0.4638.69
    • Edge (web)        • edge             • web-javascript • Microsoft Edge 95.0.1020.30

• No issues found!

**Comment by mashirozx** [comment]
Try this:

```dart
ScrollConfiguration(
  behavior: DisableGlowingOverscrollIndicator(),
  child: SmartRefresher(...),
)
```

```dart
class DisableGlowingOverscrollIndicator extends ScrollBehavior {
  @override
  Widget buildOverscrollIndicator(
      BuildContext context, Widget child, ScrollableDetails details) {
    return child;
  }
}
```

**Comment by Veeksi** [comment]
> Try this:
> 
> ```dart
> ScrollConfiguration(
>   behavior: DisableGlowingOverscrollIndicator(),
>   child: SmartRefresher(...),
> )
> ```
> 
> ```dart
> class DisableGlowingOverscrollIndicator extends ScrollBehavior {
>   @override
>   Widget buildOverscrollIndicator(
>       BuildContext context, Widget child, ScrollableDetails details) {
>     return child;
>   }
> }
> ```

Yeah, I know that workaround but then you can't see any glowing effects which is not what I want. Weird that this issue still exists.
